src/modules/lorem/index.ts

```diff
@@ -332,7 +358,26 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'number') {
+      options = { length: options };
+    }
+
+    if (options.length == null) {
+      return `${this[method]()}`;
+    }
+
+    const length = this.faker.helpers.rangeToNumber(options.length);
+
+    if (length <= 0) {
+      return '';
+    }
+
+    let text = '';
+    do {
+      text = `${text}${this[method]()} `;
+    } while (text.length <= length + 1);
+
+    return `${text.trim().substring(0, length - 1)}.`;
```

Comment:
To prevent whitespace followed by a dot:

```suggestion
    return `${text.trim().substring(0, length - 1).replace(/\s$/, '.')}.`;
```

Reply:
Alternatively, we could not deduct 1 (here and in the loop) from the length and use `.replace(/.$/, '.')` without appending the final dot.